src/main/java/org/jabref/model/openoffice/uno/UnoBookmark.java

```diff
@@ -0,0 +1,98 @@
+package org.jabref.model.openoffice.uno;
+
+import java.util.Optional;
+
+import com.sun.star.container.NoSuchElementException;
+import com.sun.star.container.XNameAccess;
+import com.sun.star.container.XNamed;
+import com.sun.star.lang.DisposedException;
+import com.sun.star.lang.IllegalArgumentException;
+import com.sun.star.lang.WrappedTargetException;
+import com.sun.star.text.XBookmarksSupplier;
+import com.sun.star.text.XTextContent;
+import com.sun.star.text.XTextDocument;
+import com.sun.star.text.XTextRange;
+
+public class UnoBookmark {
+
+    private UnoBookmark() { }
+
+    /**
+     * Provides access to bookmarks by name.
+     */
+    public static XNameAccess getNameAccess(XTextDocument doc)
+        throws
+        NoDocumentException {
+
+        XBookmarksSupplier supplier = UnoCast.unoQI(XBookmarksSupplier.class, doc);
+        try {
+            return supplier.getBookmarks();
+        } catch (DisposedException ex) {
+            throw new NoDocumentException("UnoBookmark.getNameAccess failed with" + ex);
+        }
+    }
+
+    /**
+     * Get the XTextRange corresponding to the named bookmark.
+     *
+     * @param name The name of the bookmark to find.
+     * @return The XTextRange for the bookmark, or Optional.empty().
+     */
+    public static Optional<XTextRange> getAnchor(XTextDocument doc, String name)
+        throws
+        WrappedTargetException,
+        NoDocumentException {
+
+        XNameAccess nameAccess = getNameAccess(doc);
+        return (UnoNameAccess.getTextContentByName(nameAccess, name)
+                .map(e -> e.getAnchor()));
```

Comment:
Prefer static calls in things like map(Class::method) if you don't need to pass any parameters

Reply:
ok.
At one location I could not do this:

`.map(e -> OOText.asString(e))`

Originally it was

`.map(e -> OOText.toString(e))`, but the compiler complained about `OOText::toString` being ambiguous
with `Object.toString`. Renamed to `asString` now it is ambiguous with the non-static `asString`. At least now
both are visible in OOText.java

Could rename  the non-static `asString` to `getString` if necessary. Or override `Object.toString`, 
but I do not feel that the string in the box '"textually represents"' the boxed version.

Reply:
In that case, I would leave the `toString(e)`, because this is a standard way of Java to convert to a string. This overrules the "static calls" guide :)